web/src/components/storage/ZFCPPage.tsx

```diff
@@ -47,125 +46,134 @@ import { PATHS } from "~/routes/storage";
 import { useNavigate } from "react-router-dom";
 import { inactiveLuns } from "~/utils/zfcp";
 
-/**
- * Section for zFCP controllers.
- */
-const ControllersSection = () => {
-  const controllers = useZFCPControllers();
-  const allowLUNScan = useZFCPConfig().allowLunScan;
+const LUNScanInfo = () => {
+  const { allowLunScan } = useZFCPConfig();
+  // TRANSLATORS: the text in the square brackets [] will be displayed in bold
+  const lunScanEnabled = _(
+    "Automatic LUN scan is [enabled]. Activating a controller which is \
+      running in NPIV mode will automatically configures all its LUNs.",
+  );
+  // TRANSLATORS: the text in the square brackets [] will be displayed in bold
+  const lunScanDisabled = _(
+    "Automatic LUN scan is [disabled]. LUNs have to be manually \
+      configured after activating a controller.",
+  );
 
-  const load = () => {
-    probeZFCP();
-  };
+  const msg = allowLunScan ? lunScanEnabled : lunScanDisabled;
+  const [msgStart, msgBold, msgEnd] = msg.split(/[[\]]/);
 
-  const EmptyState = () => {
-    return (
-      <Stack hasGutter>
-        <div>{_("No zFCP controllers found.")}</div>
-        <div>{_("Please, try to read the zFCP devices again.")}</div>
-        {/* TRANSLATORS: button label */}
-        <Button variant="primary" onClick={load}>
-          {_("Read zFCP devices")}
-        </Button>
-      </Stack>
-    );
-  };
-
-  const Content = () => {
-    const LUNScanInfo = () => {
-      const msg = allowLUNScan
-        ? // TRANSLATORS: the text in the square brackets [] will be displayed in bold
-          _(
-            "Automatic LUN scan is [enabled]. Activating a controller which is \
-running in NPIV mode will automatically configures all its LUNs.",
-          )
-        : // TRANSLATORS: the text in the square brackets [] will be displayed in bold
-          _(
-            "Automatic LUN scan is [disabled]. LUNs have to be manually \
-configured after activating a controller.",
-          );
-
-      const [msgStart, msgBold, msgEnd] = msg.split(/[[\]]/);
-
-      return (
-        <p>
-          {msgStart}
-          <b>{msgBold}</b>
-          {msgEnd}
-        </p>
-      );
-    };
+  return (
+    <p>
+      {msgStart}
+      <b>{msgBold}</b>
+      {msgEnd}
+    </p>
+  );
+};
 
-    return (
-      <>
-        <LUNScanInfo />
-        <ZFCPControllersTable />
-      </>
-    );
-  };
+const NoDisksFound = () => {
+  const navigate = useNavigate();
+  const controllers = useZFCPControllers();
+  const activeDisks = !controllers.some((c) => c.active);
```

Comment:
I do not like name as it emphasis that it is boolean if there is activeDisks...but it is in fact if there is any active controller.

Reply:
Ok, so please improve naming if you don't mind. I did my best looking at the code I adapted 🥲

Reply:
Nevermind, I've used `activeController` instead. You're right and now it is more meaningful and readable :)